scripts/test-util.sh

```diff
@@ -127,7 +127,7 @@ download_hypervisor_fw() {
             --header "X-GitHub-Api-Version: 2022-11-28" | grep "browser_download_url" | grep -o 'https://.*[^ "]')
     else
         echo "Using anonymous download from GitHub"
-        FW_URL=$(curl --silent https://api.github.com/repos/cloud-hypervisor/rust-hypervisor-firmware/releases/latest | grep "browser_download_url" | grep -o 'https://.*[^ "]')
+        FW_URL=$(curl --silent https://api.github.com/repos/cloud-hypervisor/rust-hypervisor-firmware/releases/latest | grep "browser_download_url" | grep -oP '"https://[^"]*hypervisor-fw"' | sed -e 's/^"//' -e 's/"$//')
```

Comment:
Upon reflection I think I would prefer to fetch the explicit version for hypervisor-fw like we do for e.g. OVMF. It gets updated relatively infrequently - as such this could be changed to simply download the required URL.

Reply:
Noted, I will change it then 🫡

Reply:
Should I change the download process like OVMF, like explicitly `wget` the pre-defined URL

Reply:
I just had a look into GitHub REST API doc: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-a-release-by-tag-name--code-samples, it needs an additional `tags` route 😥